### PR TITLE
new: debug - Preview Could Not Be Loaded inspections

### PR DESCRIPTION
**Pull request by nuritsha**

### CODE REVIEW

**Comment by arielk** [comment]
@matipojo ?

File: includes/editor.php [hunk 1]
```diff
@@ -524,6 +524,8 @@ public function enqueue_scripts() {
 			'docs_elementor_site' => 'https://go.elementor.com/docs/',
 			'help_the_content_url' => 'https://go.elementor.com/the-content-missing/',
 			'help_preview_error_url' => 'https://go.elementor.com/preview-not-loaded/',
+			'help_preview_http_error_url' => 'https://go.elementor.com/preview-not-loaded/#permissions',
```
Comment (matipojo) [inline includes/editor.php]: Move all preview related config - to `config.preview`

File: core/debug/loading-inspection-manager.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		if ( null === self::$_instance ) {
+			self::$_instance = new Loading_Inspection_Manager();
+		}
+		return self::$_instance;
+	}
+
+	/** @var Inspection_Base[] */
+	private $tests = [];
+
+	public function register_tests() {
```
Comment (matipojo) [inline core/debug/loading-inspection-manager.php]: Rename `test` to `inspection` here and future.

File: includes/preview.php [hunk 3]
```diff
@@ -273,5 +287,6 @@ public function wp_footer() {
 	 */
 	public function __construct() {
 		add_action( 'template_redirect', [ $this, 'init' ], 0 );
+		Loading_Inspection_Manager::instance()->register_tests();
```
Comment (matipojo) [inline includes/preview.php]: Move to init - after the ensure it's preview.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/editor.php (hunk 1 above)
Comment (nuritsha) [reply includes/editor.php]: done

File: includes/preview.php (hunk 3 above)
Comment (nuritsha) [reply includes/preview.php]: moved to editor.php init

File: assets/dev/js/editor/editor.js [hunk 4]
```diff
@@ -956,7 +956,7 @@ const App = Marionette.Application.extend( {
 
 		var previewWindow = this.$preview[ 0 ].contentWindow;
 
-		if ( ! previewWindow.elementorFrontend ) {
+		if ( ! previewWindow.elementorFrontend || elementor.config.debug_data.error ) {
```
[the remark this answers is not in the document]
Comment (nuritsha) [reply assets/dev/js/editor/editor.js]: done

File: core/debug/loading-inspection-manager.php (hunk 2 above)
Comment (nuritsha) [reply core/debug/loading-inspection-manager.php]: done

**Review by matipojo — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

File: includes/preview.php (hunk 3 above)
Comment (matipojo) [reply includes/preview.php]: Thanks

**Pull request merged**